frontend/src/style/styles.scss

```diff
@@ -104,3 +104,15 @@ body {
   background-color: $keyResult-attribute-bg;
   padding: 0.25em 0.75rem;
 }
+
+.default-cursor {
+  cursor: default;
+}
+
+.bg-grey {
+  background-color: $overview-bg;
+}
+
+pzsh-backdrop {
+  display: none !important;
+}
```

Comment:
Removing the backdrop is probably only a temporary solution, but the only solution I found so far